### PR TITLE
Now Creates, Reads, Updates, and Deletes

### PR DESCRIPTION
**Pull request by psihanorak**
Now Creates, Reads, Updates, and Deletes.

### CODE REVIEW

**Comment by luketlancaster** [comment]
Looks like there’s a number of merge changes in here that need to be addressed. I’d make those changes, push it up, and see if that doesn’t get rid of a bunch of the merge conflicts. If there’s still some you’ll need to Merge master into this branch too

**Review by MarkyAaronYoung — COMMENTED** [review]
You worked hard on this dude. Nice.  Luke's suggestion of fixing merge conflicts and repushing makes sense. I'll be up early if you need me to review another pr!

File: src/javascripts/components/addPlane/addPlane.js [hunk 1]
```diff
@@ -0,0 +1,43 @@
...
+    <label for="plane-name">Name:</label>
+    <input type="text" class="form-control" id="add-plane-name" placeholder="Name of Plane">
+  </div>
+  <div class="form-group">
+    <label for="plane-type">Type:</label>
+    <input type="text" class="form-control" id="add-plane-type" placeholder="Type of Plane">
+  </div>
+  <button type="submit" class="btn btn-primary" id="add-plane">Submit</button>
+</form>
+*/
```
Comment (luketlancaster) [inline src/javascripts/components/addPlane/addPlane.js]: Probably can remove this commented out code

File: src/javascripts/main.js [hunk 2]
```diff
@@ -4,6 +4,11 @@ import auth from './components/auth/auth';
 import myNavbar from './components/myNavbar/myNavbar';
 import planesList from './components/planesList/planesList';
 import authData from './helpers/data/authData';
+<<<<<<< Updated upstream
+=======
+import helpers from './helpers/clickEvents';
+
+>>>>>>> Stashed changes
```
Comment (luketlancaster) [inline src/javascripts/main.js]: Missed merge

File: src/javascripts/main.js [hunk 3]
```diff
@@ -12,7 +17,11 @@ const init = () => {
   authData.checkLoginStatus();
   auth.loginButton();
   myNavbar.logoutEvent();
+<<<<<<< Updated upstream
   planesList.showPlanes();
+=======
+  helpers.clickEvents();
+>>>>>>> Stashed changes
```
Comment (luketlancaster) [inline src/javascripts/main.js]: Here too

File: src/styles/main.scss [hunk 4]
```diff
@@ -2,7 +2,31 @@
...
+      }
+    }
+    .nav-buttons {
+      display: flex;
+      justify-content: space-evenly;
+      align-items: center;
+      align-content: center;
+      margin-bottom: 5%;
+    }
+  }
```
Comment (luketlancaster) [inline src/styles/main.scss:29]: Nesting!!! 😍😍😍

**Review by luketlancaster — CHANGES_REQUESTED** [review]
Couple of things. Looks like you’ll still need to merge master into this branch too

**Review by luketlancaster — APPROVED** [review]
Looks good to me!

**Pull request merged**